tests/integration/common/state.rs

```diff
@@ -67,21 +70,27 @@ pub struct FeeContracts {
     pub strk_fee_token_address: ContractAddress,
 }
 
+/// Helper to load cairo1 contract class and return a tuple of (name, casm, sierra) as used in
+/// TestState
+pub fn load_cairo1_classes(name: &str) -> (&str, CasmContractClass, ContractClass) {
+    (name, get_feature_casm_contract_class(name), get_feature_sierra_contract_class(name))
+}
+
 /// Fixture to create initial test state in which all test contracts are deployed.
 #[fixture]
 pub async fn initial_state(block_context: BlockContext) -> TestState {
-    let ffc = &mut FactFetchingContext::<_, PedersenHash>::new(DictStorage::default());
+    let ffc = FactFetchingContext::<_, PedersenHash>::new(DictStorage::default());
     let test_state = test_state(
         &block_context,
         BALANCE,
         get_deprecated_erc20_contract_class(),
         &[
-            ("account_with_dummy_validate", &get_deprecated_feature_contract_class("account_with_dummy_validate")),
-            ("test_contract", &get_deprecated_feature_contract_class("test_contract")),
+            ("account_with_dummy_validate", get_deprecated_feature_contract_class("account_with_dummy_validate")),
+            ("test_contract", get_deprecated_feature_contract_class("test_contract")),
         ],
         &[
-            ("account_with_dummy_validate", &get_feature_contract_class("account_with_dummy_validate")),
-            ("test_contract", &get_feature_contract_class("test_contract")),
+            // load_cairo1_classes("account_with_dummy_validate"),
+            // load_cairo1_classes("test_contract"),
```

Comment:
CC @odesenfans -- this is a workaround right now to progress further with cairo0 contracts. Uncomment this and run any of the OS tests (incl. the cairo0-only one) to see where cairo1 contracts are broken